schemas/2015-08-01/Microsoft.Network.json

```diff
@@ -35,7 +46,11 @@
                         "2015-06-15"
                     ]
                 },
+                "name": {
```

Comment:
"name" property is not necessary, and it should be under "properties" if we need it. 
All ARM resources require "type", "name", "apiVerision" properties, we have this requirement defined in deploymentTemplate.json.